src/components/LoginForm/LoginForm.tsx

```diff
@@ -40,6 +41,7 @@ export const LoginForm = () => {
             try {
                const { token } = await login(values);
                await dispatch(fetchUserByToken(token));
+               await dispatch(fetchUserLeads());
```

Comment:
**nitpick:** We should not do it here. It can be a lot of different GET requests there in Dashboard page. This current approach is similar to hardcoding, and you lose control over data fetching